static/manifest.json

```diff
@@ -1,6 +1,6 @@
 {
-  "short_name": "RSG-Group",
-  "name": "RSG-Group Webpage",
+  "short_name": "RSG",
+  "name": "RSG's Webpage",
```

Comment:
Please undo changes to this file, it will cause performance impacts in certain cases.
Anyways, it doesn't look right.

Reply:
I'll undo this